Sitback/presto
# Feature/90 media embedding support

Test steps:

- Run a full install (no need to include ecommerce).
- Add a new content (/node/add/page)
	- Add a new Text box component and embed an image using the embed media button in the WYSIWYG ("The button marked E")
		- Test this using Basic, Full and Restricted HTML format
	- Add a new Image component and add an image
	- Confirm all content being displayed properly by visiting the page
